tools/ci_static_check_reports/build_static_checkers_reports.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+from invoke import Context
+
+sys.path.insert(0, "airbyte-integrations/connectors")
+from tasks import CONFIG_FILE, TOOLS_VERSIONS, _run_task  # noqa
+
+TASK_COMMANDS: Dict[str, List[str]] = {
+    "black": [
+        f"pip install black~={TOOLS_VERSIONS['black']}",
+        f"XDG_CACHE_HOME={os.devnull} black -v {{check_option}} --diff {{source_path}}/. > {{reports_path}}/black.txt",
+    ],
+    "coverage": [
+        "pip install .",
+        f"pip install coverage[toml]~={TOOLS_VERSIONS['coverage']}",
+        "coverage xml --rcfile={toml_config_file} -o {reports_path}/coverage.xml",
+    ],
+    "flake": [
+        f"pip install mccabe~={TOOLS_VERSIONS['mccabe']}",
+        f"pip install pyproject-flake8~={TOOLS_VERSIONS['flake']}",
+        f"pip install flake8-junit-report~={TOOLS_VERSIONS['flake_junit']}",
+        "pflake8 -v {source_path} --output-file={reports_path}/flake.txt --bug-report",
+        "flake8_junit {reports_path}/flake.txt {reports_path}/flake.xml",
+        "rm -f {reports_path}/flake.txt",
+    ],
+    "isort": [
+        f"pip install colorama~={TOOLS_VERSIONS['colorama']}",
+        f"pip install isort~={TOOLS_VERSIONS['isort']}",
+        "isort -v {check_option} {source_path}/. > {reports_path}/isort.txt",
+    ],
+    "mypy": [
+        "pip install .",
+        f"pip install lxml~={TOOLS_VERSIONS['lxml']}",
+        f"pip install mypy~={TOOLS_VERSIONS['mypy']}",
+        "mypy {source_path} --config-file={toml_config_file} --cobertura-xml-report={reports_path}",
+    ],
+    "test": [
+        "mkdir {venv}/source-acceptance-test",
+        "cp -f $(git ls-tree -r HEAD --name-only {source_acceptance_test_path} | tr '\n' ' ') {venv}/source-acceptance-test",
+        "pip install build",
+        f"python -m build {os.path.join('{venv}', 'source-acceptance-test')}",
+        f"pip install {os.path.join('{venv}', 'source-acceptance-test', 'dist', 'source_acceptance_test-*.whl')}",
+        "[ -f requirements.txt ] && pip install -r requirements.txt 2> /dev/null",
+        "pip install .",
+        "pip install .[tests]",
+        "pip install pytest-cov",
+        "pytest -v --cov={source_path} --cov-report xml:{reports_path}/pytest.xml {source_path}/unit_tests",
+    ],
+}
+
+
+def build_static_checkers_reports(modules: list) -> None:
+    os.chdir(Path(__file__).parents[2])
+    ctx = Context()
+    toml_config_file = os.path.join(os.getcwd(), "pyproject.toml")
+
+    for module_path in modules:
+        reports_path = f"{os.getcwd()}/static_checker_reports/{module_path}"
+        if not os.path.exists(reports_path):
+            os.makedirs(reports_path)
+
+        for checker in TASK_COMMANDS.keys():
+            _run_task(
```

Comment:
the method `keys` is not required for working with dict names. You can make same without it:
```
for checker in TASK_COMMANDS:
   .....
```

Reply:
Oh, well, it's more clear style, in my opinion
I'll check what is more used in our codebase